conda-forge.yml

```diff
@@ -4,3 +4,6 @@ travis:
 appveyor:
   secure:
     BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
+compiler_stack: comp7
```

Comment:
We need this to upload the package to the `gcc7` channel. There is no compilation part here but the recipe is "arch" in a way due to the patch differences in the platforms.

I'm not sure why the migrator did not catch these recipes. Maybe @CJ-Wright may help here.

PS: another alternative would be to "copy" these to the `gcc7` label without a rebuild.

Reply:
Thanks, Filipe. I'll make the same change to DCW as well and bumb the build number. Should be quick to rebuild these.

Reply:
I'm not certain why the migrator didn't PR this at the beginning, maybe because it doesn't have requirements?